spec/02-integration/05-proxy/01-router_spec.lua

```diff
@@ -134,6 +134,26 @@ describe("Router", function()
       assert.equal("api-1", res.headers["kong-api-name"])
     end)
 
+    it("routes by method-only if no Host header present", function()
```

Comment:
Would it be possible to create a new `describe` block for empty/no Host header tests? Like so:
```lua
describe("requests with no Host header", function()
  it("HTTP/1.0 routes normally", function()

  end)

  it("HTTP/1.1 is rejected by NGINX", function()
    -- new test: assert NGINX's behavior with HTTP/1.1 (missing from this PR)
  end)
end)
```